README.md

```diff
@@ -40,6 +40,8 @@ cd uno
 eval "$(./bin/uno env)"             # Bash-specific command that sets up current shell
```

Comment:
The new sentence about verification made me think of running the Accumulo shell after eval.